virt/virt-2-4-release-notes.adoc

```diff
@@ -54,6 +54,9 @@ include::modules/virt-what-you-can-do-with-virt.adoc[leveloffset=+2]
 == Notable technical changes
 //Placeholder for new content.
 
+//CNV-5465-Disconnected installation
+* {VirtProductName} can be installed on disconnected clusters in restricted network environments that do not have Internet connectivity. You can create local mirrors for the {VirtProductName} operator and install from a local Operator catalog image that is accessible to the disconnected cluster. For information on restricted network environments, see the xref:../operators/olm-restricted-networks.adoc#olm-understanding-operator-catalog-images_olm-restricted-networks[restricted networks] chapter.
```

Comment:
This is looking good, but I have a few nits:

- s/{VirtProductName} operator/{VirtProductName} Operator
- s/and install from a local Operator catalog image/and install the Operator from a local catalog image
- I don't think "chapter" accurately describes a modular docs section. You're also not just talking about restricted network environments, but rather OLM on restricted networks. I would change the xref to something like this: 
`** Learn more about xref:../operators/olm-restricted-networks.adoc#olm-understanding-operator-catalog-images_olm-restricted-networks[using Operator Lifecycle Manager on restricted networks].`

Reply:
HI @ousleyp I'll make the modifications and update you shortly.

Reply:
@ousleyp Peer review edits were incorporated. Please review.